src/library_wasmfs.js

```diff
@@ -59,9 +59,12 @@ FS.createPreloadedFile = FS_createPreloadedFile;
 #endif
     'malloc',
     'free',
+    'wasmfs_create_jsimpl_backend',
+    '$wasmFS$backends'
 #endif
   ],
   $FS : {
+    devices: {},
```

Comment:
It would be better to keep this off on the side, so it isn't included unnecessarily (as it is, all uses of `FS` will get it, even if they didn't enable `FORCE_FILESYSTEM` for example). See e.g. `wasmFSPreloadedFiles` in this file for an example of such a property.